Exam/CR/src/exercice1/Polynom.java

```diff
@@ -0,0 +1,44 @@
+package exercice1;
+
+public class Polynom {
+	
+	private int taille;
+	private double[] coeffs;
+	
+	public Polynom(double[] coeffs) {
+		//constructeur
+		this.taille = coeffs.length;
+		this.coeffs = coeffs;
+	}
+	
+	public double eval(double x) {
+		double result = 0;
+		for(int i= 0; i <= taille - 1; i++) {
+			if(i == 0)
+				result = coeffs[i] + result;
+			else
+				result = coeffs[i] * Math.pow(x, i) + result; 
+		}
+		return result;
+	}
+	
+	public Polynom primitive() {
+		int i = 0;
+		double[] coeffPolyTest = {0,0,1};
+		double[] coeffCreer = new double[coeffPolyTest.length + 1];
+		coeffCreer[0] = 0;
+		for(i = 0; i<coeffPolyTest.length; ++i) {
```

Comment:
You just needed to replace `coeffPolyTest` with `(this).coeffs` and you were good...